src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -1304,10 +1305,22 @@ public void Bootstrap(string selfUrl, bool forNewCluster = false)
                     "A"
                 );
 
+                
+
                 SetTopology(null, ctx, topology);
 
                 SwitchToSingleLeader(ctx);
 
+                // If there are trusted certificates in the local state, we will register them in the cluster now
+                foreach (var localCertKey in serverStore.Cluster.GetCertificateKeysFromLocalState(ctx))
+                {
```

Comment:
Need to register my cert there too
Need to also delete these when adding new node